grouper/fe/template_util.py

```diff
@@ -7,6 +10,29 @@
 from grouper.fe.settings import settings
 
 
+RESOURCE_INTEGRITY_VALUES = {}
+CHUNK_SIZE = 1024 * 16
+
+
+def get_integrity_of_static(path):
```

Comment:
Please cache this.

Reply:
It is cached in the `RESOURCE_INTEGRITY_VALUES` hash